java/viewshed-geoelement/src/main/java/com/esri/arcgisruntime/sample/viewshedgeoelement/MainActivity.java

```diff
@@ -0,0 +1,268 @@
+import com.esri.arcgisruntime.mapping.view.OrbitGeoElementCameraController;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.ModelSceneSymbol;
+import com.esri.arcgisruntime.symbology.Renderer;
+import com.esri.arcgisruntime.symbology.SceneSymbol;
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+
+public class MainActivity extends AppCompatActivity {
+
+  private static final String TAG = MainActivity.class.getSimpleName();
+
+  private static final LinearUnit METERS = new LinearUnit(LinearUnitId.METERS);
+  private static final AngularUnit DEGREES = new AngularUnit(AngularUnitId.DEGREES);
+  private SceneView mSceneView;
+  private Point mWaypoint;
+  private Graphic mTankGraphic;
+  private Timer mTimer;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // create a scene and add a basemap to it
+    ArcGISScene scene = new ArcGISScene();
+    scene.setBasemap(Basemap.createImagery());
+
+    // add the SceneView to the stack pane
+    mSceneView = findViewById(R.id.sceneView);
+    mSceneView.setScene(scene);
+
+    // add base surface for elevation data
+    Surface surface = new Surface();
+    surface.getElevationSources().add(new ArcGISTiledElevationSource(getString(R.string.elevation_service)));
+    scene.setBaseSurface(surface);
+
+    // add a scene layer
+    ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer(getString(R.string.buildings_layer));
+    scene.getOperationalLayers().add(sceneLayer);
+
+    // request read permission
+    requestWritePermission();
+  }
+
+  /**
+   * Creates a GeoElement Viewshed fixed to a graphic of a tank. Includes a touch listener which uses a single tap as a
+   * waypoint for navigation of the tank and associated viewshed.
+   */
+  private void viewshedGeoElement() {
+
+    // load tank model from assets into cache directory
+    copyFileFromAssetsToCache(getString(R.string.bradley_model));
+    copyFileFromAssetsToCache(getString(R.string.bradley_skin));
+
+    // create a graphics overlay for the tank
+    GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
+    graphicsOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.RELATIVE);
+    mSceneView.getGraphicsOverlays().add(graphicsOverlay);
+
+    // set up heading expression for tank
+    SimpleRenderer renderer3D = new SimpleRenderer();
+    Renderer.SceneProperties renderProperties = renderer3D.getSceneProperties();
+    renderProperties.setHeadingExpression("[HEADING]");
+    graphicsOverlay.setRenderer(renderer3D);
+
+    String pathToModel = getCacheDir() + File.separator + getString(R.string.bradley_model);
+
+    ModelSceneSymbol tankSymbol = new ModelSceneSymbol(pathToModel, 10.0);
+    tankSymbol.setHeading(90);
+    tankSymbol.setAnchorPosition(SceneSymbol.AnchorPosition.BOTTOM);
+    tankSymbol.loadAsync();
```

Comment:
Is there a reason why the tank symbol is explicitly loaded? Will this not auto-load?